post-microsservice/src/main/kotlin/com/example/postmicrosservice/controller/PostController.kt

```diff
@@ -9,5 +10,18 @@ import org.springframework.web.bind.annotation.RestController
 class PostController(
     private val postService: PostService
 ) {
+    @PostMapping("/create")
+    fun create(post: Post){
+        postService.createPost(post)
+    }
 
+    @PutMapping("/update/{id}")
+    fun update(@PathVariable id: Long, post: Post){
+        postService.updatePost(id, post)
+    }
+        
+    @DeleteMapping("/delete/{id}")
+    fun delete(@PathVariable id: Long){
+        postService.deletePost(id)
+    }
 }
```

Comment:
WDYT about doing the test too? 
I don't have a example of an Controller test here, but you can do something like the Service Test mocking the response and assert it. If you need any help we can do this together.